shared/test/test_assets.rb

```diff
@@ -182,6 +183,40 @@ def test_viewing_abusive_assets
     @api.delete_object(asset_name)
   end
 
+  def test_assets_copy_some
+    delete_all_assets('assets_test/1/2')
+    dest_channel_id = create_channel
+    src_api = FilesApiTestHelper.new(current_session, 'assets', @channel_id)
+    dest_api = FilesApiTestHelper.new(current_session, 'assets', dest_channel_id)
+
+    image_filename = 'çat.jpg'
+    image_body = 'stub-image-contents'
+
+    sound_filename = 'woof.mp3'
+    sound_body = 'stub-sound-contents'
+
+    _, image_filename = post_asset_file(src_api, image_filename, image_body, 'image/jpeg')
+    _, sound_filename = post_asset_file(src_api, sound_filename, sound_body, 'audio/mpeg')
+    src_api.patch_abuse(10)
```

Comment:
We clear the abuse score on remix because that's the way to "fix" an abusive project (do a remix, fix stuff, then share your new link). Probably we shouldn't be able to import/export abusive projects.